aaad/aaad.py

```diff
@@ -21,7 +21,6 @@
 secret_key = os.getenv('SECRET_KEY', 'test_key_NOT_a_secret')
 session_id_key = os.getenv('SESSION_ID_KEY', 'aaadsid')
 session_id_domain = os.getenv('SESSION_ID_DOMAIN', None)
-htpasswd_file = os.getenv('HTPASSWD_FILE', '')
 
 class AuthHTTPServer(ThreadingMixIn, HTTPServer, ):
     pass
```

Comment:
The new code replaces this whole module. 
I'll just replace this file with the new one (aaad_app.)